src/Microsoft.ML.Data/Transforms/FeatureContributionCalculationTransformer.cs

```diff
@@ -312,6 +312,10 @@ public static class Defaults
             _predictor = modelParameters;
         }
 
+        /// <summary>
+        /// Returns the <see cref="SchemaShape"/> of the schema which will be produced by the transformer.
+        /// Used for schema propagation and verification in a pipeline.
+        /// </summary>
```

Comment:
is it same as description in base method? you can use 
` ///<inheritdoc/> ` instead